en/can/README.md

```diff
@@ -0,0 +1,88 @@
+# CAN
+
+[Controller Area Network (CAN)](https://en.wikipedia.org/CAN_bus) is a robust wired network that allows drone components such as flight controller, ESCs, sensors, and other peripherals, to communicate with each other.
+Because it is designed to be democratic and uses differential signaling, it is very robust even over longer cable lengths (on large vehicles), and avoids a single point of failure.
+CAN also allows status feedback from peripherals and convenient firmware upgrades over the bus.
+
+PX4 supports two software protocols for communicating with CAN devices:
+
+- [DroneCAN](../dronecan/README.md): PX4 recommends this for most common setups.
+  It is well supported by PX4, is a mature product with extensive peripheral support, and has had years of testing.
+- [Cyphal](https://opencyphal.org): PX4 support is a "work in progress".
+  Cyphal is a much newer protocol which allows more flexibility and configuration, especially on larger and more complex vehicles.
+  It has not yet seen significant adoption.
+
+:::note
+Both DroneCAN and Cyphal originate from an earlier project named UAVCAN.
```

Comment:
Note, I'm slightly hiding the community disagreement here. It's obvious something happened, but irrelevant other than for people to know the mapping